web/server/vue-cli/src/views/NewFeatures.vue

```diff
@@ -2,6 +2,548 @@
   <!-- eslint-disable max-len -->
   <v-container fluid>
     <v-timeline align-top>
+      <v-timeline-item fill-dot icon="mdi-star" color="green lighten-1">
+        <new-release-item color="green lighten-1">
+          <template v-slot:title>
+            <a
+              href="https://github.com/Ericsson/codechecker/releases/tag/v6.13.0"
+              target="_blank"
+              class="white--text"
+            >
+              Highlights of CodeChecker 6.13.0 release
+            </a>
+          </template>
+
+          <new-feature-item>
+            <template v-slot:title>
+              New web UI
+            </template>
+            In this release the UI framework was completely replaced to
+            increase usability, stability and performance. <br>
+            The new framework allows a lot of improvements like:
+            <ul>
+              <li>faster page load</li>
+              <li>faster navigation</li>
+              <li>improved front-end testing</li>
+              <li>less load on the server</li>
+            </ul>
+            With the new UI the permalinks are backward compatible so the saved
+            URLs should work as before. Additionally to the UI improvements
```

Comment:
I think a line break before the `Additionally`  and the `Note!` words would improve the readability.